docs/faq.rst

```diff
@@ -3,9 +3,123 @@ FAQ
 
 Here you'll find answers to some frequently asked questions.
 
+Breaking changes in tox 4
+-------------------------
+Version 4 of tox should be mostly backwards compatible with version 3, with the following exceptions:
+
+tox 4 - Python support
```

Comment:
using the `tox 4` prefix for every sub-heading seems redundant...  I think the `Breaking changes in tox 4` section header is enough.

Reply:
I disagree, this keeps the generated url anchors relevant.